### DISCUSSION TITLE
Other sources ?

### DISCUSSION THREAD

**Discussion by 2Fox2** [kodjodevf/mangayomi#17] [Q&A]
Hi, I'm new to GitHub and I saw this very nice application. I'd like to know if I can add other sources or modify them, and if so, can you explain ? 

Sorry if my question is a bit silly and also sorry for my English. 

@kodjodevf

**Comment by kodjodevf** [comment]
To learn how to add other sources, please read this [DOC](https://github.com/kodjodevf/mangayomi-extensions/blob/main/CONTRIBUTING.md)
